r/vignettes/dataset.Rmd

```diff
@@ -56,7 +56,7 @@ you may need to increase R's download timeout from the default of 60 seconds, e.
 
 ```{r, eval = FALSE}
 bucket <- "https://voltrondata-labs-datasets.s3.us-east-2.amazonaws.com"
-for (year in 2009) {
+for (year in 2009:2022) {
   if (year == 2022) {
     # We only have through June 2019 there
```

Comment:
Sorry, to nit-pick here, but the comment on line 61 "We only have through June 2019 there" should now say whatever month in 2022, otherwise it's confusing.

Reply:
oh yes this is good point.